lib/money/money.rb

```diff
@@ -1,77 +1,107 @@
-require 'bigdecimal'
-require 'bigdecimal/util'
-
 class Money
   include Comparable
 
-  attr_reader :value, :cents
+  attr_reader :value, :cents, :currency
 
   class << self
-    def new(value = 0, _currency = nil)
+    attr_accessor :parser, :default_currency
```

Comment:
added instructions in the readme on best practices. `Money.default_curency = Money::Currency.new('usd')` should be used in an initializer and not in the scope of a request